HW2_Trofimov/calculator.py

```diff
@@ -0,0 +1,42 @@
+def main():
+    calc_input = input()
+    calc_input_lst = calc_input.split()
+
+    operation = calc_input_lst[1]
+    number_1 = calc_input_lst[0]
+    number_1 = float(number_1)
+    number_2 = calc_input_lst[2]
+    number_2 = float(number_2)
```

Comment:
В целом код рабочий, хоть и не очень питонячий. Особенно переменные типа calc_input и  выглядят calc_input_lst громоздко (хотя нейминг по факту ок!). Это ничего страшного, мы с вами еще научимся опитонячивать наш код. Например, тут это можно было бы сделать так:
```suggestion
    num1, operator, num2 = input().split()
    num1, num2 = float(num1), float(num2)
```

Reply:
трюк со сплитом инпута в три переменные хорош...